microsoft-edge/webview2/concepts/end-user-faq.md

```diff
@@ -0,0 +1,70 @@
+---
+title: WebView2 end-user FAQ
+description: Information for end users about WebView2 processes, memory and CPU usage, and automatic installation.
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.technology: webview
+ms.date: 03/30/2023
+---
+# WebView2 end-user FAQ
+
+This FAQ is for end users, to explain the `msedgewebview2.exe` application on your computer.  If your question isn't listed here, you can click the **Feedback** button in the top-right corner, to enter a question as an [Issue in the edge-developer Docs repo](https://github.com/MicrosoftDocs/edge-developer/issues), and your question might be added to this article.  See also [Contact the WebView2 Team](../contact.md).
+
+
+<!-- ====================================================================== -->
+## What is WebView2?
+
+WebView2 is a way for app developers to embed web content (such as HTML, JavaScript, and CSS) in Windows applications. By including the WebView2 control in an app, a developer can write code for a website or web app, and then reuse that web code in their Windows application, saving time and effort.  See [Introduction to Microsoft Edge WebView2](../index.md).
+
+
+<!-- ====================================================================== -->
+## What's the purpose of the WebView2 executable?
+
+WebView2 (`msedgewebview2.exe`) is a Microsoft product that is updated automatically (by using the _Evergreen_ distribution method) to ensure that your applications have the latest feature support and security fixes.
+
+Most people will see WebView2 used by Widgets, Teams, Office, Outlook, Weather, and many more applications. Any application can use WebView2.
+
+
+<!-- ====================================================================== -->
+## What are the WebView2 processes that are running?
+
+WebView2 follows the process model that's used in the Chromium browser engine of Microsoft Edge, described in [Process Model and Site Isolation](https://chromium.googlesource.com/chromium/src/+/main/docs/process_model_and_site_isolation.md).  Having functionality broken up into multiple processes help reliability, security, and performance.  Each process performs a certain responsibility, and can do that work uninterrupted, even if one of the other processes hits a problem.
+
+There are normally a few processes:
+*  The WebView2 manager.
+*  A GPU process.
+*  Utility processes, such as network or audio, depending on the content.
+*  Renderer processes.
+
+Your machine will have one set of processes for each app that uses WebView2, and typically one renderer process for each WebView2 control that's in the app, similar to how you have one renderer process per tab in a browser.  For details, see [Process model for WebView2 apps](/microsoft-edge/webview2/concepts/process-model).
```

Comment:
Consider adding a screenshot of the Task Manager showing an app that uses WebView2, with its process expanded to show the various processes listed above.

Reply:
@captainbrosset - pls in VS open Win32 GetStart, build, run, capture in Task Manager to replace the pngs below.

@aluhrs13 

gh rendered section:
https://github.com/mikehoffms/edge-developer/blob/user/mikehoffms/end-user-faq/microsoft-edge/webview2/concepts/end-user-faq.md#what-memory-and-cpu-resources-does-webview2-use

Reply:
We should note with screenshots that Windows 10 and early versions of Windows 11 have pretty mediocre views. New Win11 builds have details.

Reply:
"In Windows 10 and early versions of Windows 11, the **Processes** tab only lists the name of the app that uses WebView2:

In the latest Windows 11 versions, the **Processes** tab lists each WebView2 process:"